packages/tenant-event-consumer/src/handlers/messageHandlerV1.ts

```diff
@@ -0,0 +1,57 @@
+import { TenantEventV1 } from "@pagopa/interop-outbound-models";
+import { Logger, AppContext } from "pagopa-interop-tracing-commons";
+import { P, match } from "ts-pattern";
+import { config } from "../utilities/config.js";
+import { OperationsService } from "../services/operationsService.js";
+import {
+  correlationIdToHeader,
+  kafkaMessageMissingData,
+} from "pagopa-interop-tracing-models";
+
+export async function handleMessageV1(
+  event: TenantEventV1,
+  operationsService: OperationsService,
+  ctx: AppContext,
+  logger: Logger,
+): Promise<void> {
+  await match(event)
+    .with(
+      {
+        type: P.union("TenantCreated", "TenantUpdated"),
+      },
+      async (evt) => {
+        if (!evt.data.tenant) {
+          throw kafkaMessageMissingData(config.kafkaTopic, event.type);
+        }
+
+        const { tenant } = evt.data;
+
+        await operationsService.saveTenant(
+          { ...correlationIdToHeader(ctx.correlationId) },
+          {
+            tenantId: tenant.id,
+            externalId: tenant.externalId?.value || "TODO",
+            origin: tenant.externalId?.origin || "TODO",
+            name: tenant.name || "TODO",
```

Comment:
Make sure that are not mandatory so sql can be updated to allow null

Reply:
Ok, let's mark the columns to "nullable"